src/python/pants/backend/rust/subsystems/rust.py

```diff
@@ -16,20 +17,24 @@ class RustSubsystem(Subsystem):
     toolchain = StrOption(
         "--toolchain",
         default="stable",
-        help=(
-            "Name of a Rust toolchain to use for all builds. The toolchain name will be provided to "
-            "Rustup to find the Toolchain."
+        help=help_text(
+            """
+            Name of a Rust toolchain to use for all builds. The toolchain name will be provided to
+            Rustup to find the Toolchain.
+            """
         ),
     )
 
     _rustup_search_paths = StrListOption(
         "--rustup-search-paths",
         default=["<PATH>"],
-        help=(
-            "A list of paths to search for Rustup.\n\n"
-            "Specify absolute paths to directories with the `rustup` binary, e.g. `/usr/bin`. "
-            "Earlier entries will be searched first.\n\n"
-            "The special string '<PATH>' will expand to the contents of the PATH env var."
+        help=help_text(
+            """
+            A list of paths to search for Rustup.
+            Specify absolute paths to directories with the `rustup` binary, e.g. `/usr/bin`.
```

Comment:
You still use two newlines, kinda like markdown

Reply:
I do not understand the above comment

Reply:
The earlier version had two newlines in places. For example, `A list of paths to search for Rustup.\n\n`. You should still include the two newlines but just as multiple lines:

```
help=help_text(
    """
    A list of paths to search for Rustup.

    Specify absolute paths to directories with the `rustup` binary, e.g. `/usr/bin`. 
    """
)
```

Similar to what is done in the `help_text` in `RustPackageTarget`.